portfolio/src/main/webapp/script.js

```diff
@@ -0,0 +1,23 @@
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+fetch('/data').then(response => response.json()).then((comment) => {
+	let doc = document.getElementById('comment-container');
+	for(x in comment){
+		doc.innerHTML += "<li>";
+		doc.innerHTML += "<h3>" + comment[x].name + "</h3>";
+		doc.innerHTML += "<p>" + comment[x].content + "</p>";
+		doc.innerHTML += "</li>";
```

Comment:
Take a look at JavaScript template literals to make this a little easier to read: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals. That will let you add the HTML in a single statement.